docs/pages/eas-build-custom-builds/custom-build-config.mdx

```diff
@@ -103,9 +106,28 @@ Let's create a new [build profile](/build/eas-json/#build-profiles) called `test
   "build": {
     /* @hide ... */ /* @end */
     "test": {
+      "config": "test.yml",
+      "withoutCredentials": true
+    },
+}
+```
+
+We can also use the `withoutCredentials` option to skip the credentials setup on the CLI side as they are not needed for running tests.
```

Comment:
```suggestion
You can also use the `withoutCredentials` option to skip the credentials setup on the CLI side, as they are not needed for running tests.
```

[Let's phrase this sentence in second person voice](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#write-in-the-second-person)